.github/workflows/visual-regression-tests.yml

```
@@ -0,0 +1,23 @@
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  visual-regression-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # Also runs `bundle install`
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "12"
+          cache: yarn
+- run: yarn install --frozen-lockfile
+- uses: percy/exec-action@v0.3.1
+  with:
+    command: bundle exec rspec --tag visual_regression
```

I think that this repo doesn't use RSpec for tests, it uses Minitest (at least, that's what it looks like in the `Rakefile`) - which could be why this command isn't doing anything.

Or because none of the tests are tagged with `visual_regression` nothing is being found to run.

I think this command will need to be changed to
```suggestion
    command: bundle exec rake test_unit
```

but that will run _all_ the tests, so we could either 

 - quick and dirty - only run the homepage test for the moment (which still runs all the other non-visual regression tests at the same time)
 - less tech debty but more work - create a folder specifically for adding only visual regression tests, which could be ignored for local runs and only run in this action.

---

Gonna quick and dirty it first to make sure it works, then another PR/issue to improve it?